packages/assets-controllers/src/token-prices-service/codefi-v2.ts

```diff
@@ -0,0 +1,344 @@
+  // Polkadot
+  'dot',
+  // Yearn.finance
+  'yfi',
+  // US Dollar
+  'usd',
+  // United Arab Emirates Dirham
+  'aed',
+  // Argentine Peso
+  'ars',
+  // Australian Dollar
+  'aud',
+  // Bangladeshi Taka
+  'bdt',
+  // Bahraini Dinar
+  'bhd',
+  // Bermudian Dollar
+  'bmd',
+  // Brazil Real
+  'brl',
+  // Canadian Dollar
+  'cad',
+  // Swiss Franc
+  'chf',
+  // Chilean Peso
+  'clp',
+  // Chinese Yuan
+  'cny',
+  // Czech Koruna
+  'czk',
+  // Danish Krone
+  'dkk',
+  // Euro
+  'eur',
+  // British Pound Sterling
+  'gbp',
+  // Hong Kong Dollar
+  'hkd',
+  // Hungarian Forint
+  'huf',
+  // Indonesian Rupiah
+  'idr',
+  // Israeli New Shekel
+  'ils',
+  // Indian Rupee
+  'inr',
+  // Japanese Yen
+  'jpy',
+  // South Korean Won
+  'krw',
+  // Kuwaiti Dinar
+  'kwd',
+  // Sri Lankan Rupee
+  'lkr',
+  // Burmese Kyat
+  'mmk',
+  // Mexican Peso
+  'mxn',
+  // Malaysian Ringgit
+  'myr',
+  // Nigerian Naira
+  'ngn',
+  // Norwegian Krone
+  'nok',
+  // New Zealand Dollar
+  'nzd',
+  // Philippine Peso
+  'php',
+  // Pakistani Rupee
+  'pkr',
+  // Polish Zloty
+  'pln',
+  // Russian Ruble
+  'rub',
+  // Saudi Riyal
+  'sar',
+  // Swedish Krona
+  'sek',
+  // Singapore Dollar
+  'sgd',
+  // Thai Baht
+  'thb',
+  // Turkish Lira
+  'try',
+  // New Taiwan Dollar
+  'twd',
+  // Ukrainian hryvnia
+  'uah',
+  // Venezuelan bolívar fuerte
+  'vef',
+  // Vietnamese đồng
+  'vnd',
+  // South African Rand
+  'zar',
+  // IMF Special Drawing Rights
+  'xdr',
+  // Silver - Troy Ounce
+  'xag',
+  // Gold - Troy Ounce
+  'xau',
+  // Bits
+  'bits',
+  // Satoshi
+  'sats',
+] as const;
+
+/**
+ * A currency that can be supplied as the `vsCurrency` parameter to
+ * the `/spot-prices` endpoint. Covers both uppercase and lowercase versions.
+ */
+type SupportedCurrency =
+  | (typeof SUPPORTED_CURRENCIES)[number]
+  | Uppercase<(typeof SUPPORTED_CURRENCIES)[number]>;
+
+/**
+ * The list of chain IDs that can be supplied as the `vsCurrency` parameter to
+ * the `/spot-prices` endpoint, but in hexadecimal form (for consistency with
+ * how we represent chain IDs in other places).
+ */
+export const SUPPORTED_CHAIN_IDS = [
+  // Ethereum Mainnet
+  '0x1',
+  // OP Mainnet
+  '0xa',
+  // Cronos Mainnet
+  '0x19',
+  // BNB Smart Chain Mainnet
+  '0x38',
+  // Syscoin Mainnet
+  '0x39',
+  // OKXChain Mainnet
+  '0x42',
+  // Hoo Smart Chain
+  '0x46',
+  // Meter Mainnet
+  '0x52',
+  // TomoChain
+  '0x58',
+  // Gnosis
+  '0x64',
+  // Velas EVM Mainnet
+  '0x6a',
+  // Fuse Mainnet
+  '0x7a',
+  // Huobi ECO Chain Mainnet
+  '0x80',
+  // Polygon Mainnet
+  '0x89',
+  // Fantom Opera
+  '0xfa',
+  // Boba Network
+  '0x120',
+  // KCC Mainnet
+  '0x141',
+  // zkSync Era Mainnet
+  '0x144',
+  // Theta Mainnet
+  '0x169',
+  // Metis Andromeda Mainnet
+  '0x440',
+  // Moonbeam
+  '0x504',
+  // Moonriver
+  '0x505',
+  // Base
+  '0x2105',
+  // Shiden
+  // NOTE: This is the wrong chain ID, this should be 0x150
```

Comment:
I'm asking Tomas about this: https://github.com/MetaMask/metamask-mobile/pull/7957#discussion_r1412567673